tests/system-tests/ran-du/tests/stability-no-workload.go

```diff
@@ -0,0 +1,106 @@
+	"github.com/openshift-kni/eco-gotests/tests/system-tests/internal/stability"
+	. "github.com/openshift-kni/eco-gotests/tests/system-tests/ran-du/internal/randuinittools"
+)
+
+var _ = Describe(
+	"StabilityNoWorkload",
+	Ordered,
+	ContinueOnFailure,
+	Label("StabilityNoWorkload"), func() {
+		var (
+			clusterName string
+			err         error
+		)
+		BeforeAll(func() {
+
+			By("Fetching Cluster name")
+			clusterName, err = platform.GetOCPClusterName(APIClient)
+			Expect(err).ToNot(HaveOccurred(), "Failed to get cluster name")
+
+		})
+		It("StabilityNoWorkload", reportxml.ID("TBD"), Label("StabilityNoWorkload"), func() {
+
+			outputDir := RanDuTestConfig.StabilityOutputPath
+			policiesOutputFile := fmt.Sprintf("%s/stability_policies.log", outputDir)
+			ptpOutputFile := fmt.Sprintf("%s/stability_ptp.log", outputDir)
+			namespaces := []string{"openshift-etcd", "openshift-apiserver"}
+
+			totalDuration := time.Duration(RanDuTestConfig.StabilityDurationMins) * time.Minute
+			interval := time.Duration(RanDuTestConfig.StabilityIntervalMins) * time.Minute
+			startTime := time.Now()
+
+			By("Start collecting metrics during the stability test duration defined")
+			for time.Since(startTime) < totalDuration {
+
+				if RanDuTestConfig.PtpEnabled {
+					err = stability.SavePTPStatus(APIClient, ptpOutputFile, interval)
+					if err != nil {
+						fmt.Printf("Error, could not save PTP")
+					}
+				}
+
+				err = stability.SavePolicyStatus(APIClient, clusterName, policiesOutputFile)
+				if err != nil {
+					fmt.Printf("Error, could not save policies status")
+				}
+
+				for _, namespace := range namespaces {
+					err = stability.SavePodsRestartsInNamespace(APIClient,
+						namespace, fmt.Sprintf("%s/stability_%s.log", outputDir, namespace))
+					if err != nil {
+						fmt.Printf("Error, could not save Pod restarts")
+					}
+
+				}
+
+				time.Sleep(interval)
+			}
+
+			// Final check of all values
+			By("Check all results")
+			var stabilityErrors []string
+
+			// Verify policies
+			By("Check Policy changes")
+			_, err := stability.VerifyStabilityStatusChange(policiesOutputFile)
+			if err != nil {
+				stabilityErrors = append(stabilityErrors, err.Error())
+			}
+
+			// Verify podRestarts
+			By("Check Pod restarts")
+			for _, namespace := range namespaces {
+				_, err := stability.VerifyStabilityStatusChange(fmt.Sprintf("%s/stability_%s.log", outputDir, namespace))
+				if err != nil {
+					stabilityErrors = append(stabilityErrors, err.Error())
+				}
+			}
+
+			// Verify PTP output
+			By("Check PTP results")
+			if RanDuTestConfig.PtpEnabled {
+				_, err = stability.VerifyStabilityStatusChange(ptpOutputFile)
+				if err != nil {
+					stabilityErrors = append(stabilityErrors, err.Error())
+				}
+			}
+
+			By("Check if there been any error")
+			if len(stabilityErrors) > 0 {
+				Expect(stabilityErrors).ToNot(HaveOccurred(), "One or more errors in stability tests")
```

Comment:
Can we also print the stabilityErrors to make it easier to understand the errors in case of failure?

Reply:
Sure, done.